frontend/src/images/icons/NotebookIcon.ts

```diff
@@ -0,0 +1,13 @@
+import { createIcon } from '@patternfly/react-icons/dist/esm/createIcon';
+
+const NotebookIcon = createIcon({
```

Comment:
According to the [mocks](https://www.figma.com/design/xejiiAIdT9ezaBKCNTss6b/Projects-table-view?node-id=89-113418&node-type=frame&t=YJhlCks2RUKXaDas-0), it looks like this icon should be solid. Wondering if we can update the svg path here? WDYT @jeff-phillips-18

Reply:
@jenny-s51 You're looking at the old mocks, In the updated[mocks](https://www.figma.com/design/xejiiAIdT9ezaBKCNTss6b/Projects-table-view?node-id=1521-112631&t=V9wgRDW5KiryjAqJ-1) (which are ready for dev), we're using the outlined icon, just like Jeff added.

One small difference is that the icon appears slightly smaller than it should be (compared to the mocks). 
@jeff-phillips-18 Could you take a look at this?

Reply:
The size in the table is based on the font size (14px) while the mocks show 16px. I have updated the code to set the icons size to 16px. Screen shot above has been updated.